src/cloudant/design_document.py

```diff
@@ -243,3 +207,128 @@ def info(self):
         GET databasename/_design/{ddoc}/_info
         """
         raise NotImplementedError("_info not yet implemented")
+
+class CloudantDesignDocument(DesignDocument):
+    """
+    Encapsulates a Cloudant version of a
+    :class:`~cloudant.design_document.DesignDocument`.
+    A CloudantDesignDocument object is instantiated with a reference
+    to a Cloudant database and provides an API to query index management,
```

Comment:
Change `Cloudant database` to a link to CloudantDatabase use 2 lines above DesignDocument as a reference.